src/routes/ViewAmendmentRoute.js

```diff
@@ -1,25 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
 import compose from 'compose-function';
 import { FormattedMessage } from 'react-intl';
 
 import SafeHTMLMessage from '@folio/react-intl-safe-html';
 import { CalloutContext, stripesConnect } from '@folio/stripes/core';
 import { ConfirmationModal } from '@folio/stripes/components';
+import { preventResourceRefresh } from '@folio/stripes-erm-components';
 import DuplicateAmendmentModal from '../components/DuplicateAmendmentModal';
 
 import withFileHandlers from './components/withFileHandlers';
 import View from '../components/Amendment';
 
 import { errorTypes } from '../constants';
 
+const RECORDS_PER_REQUEST = 100;
+
 class ViewAmendmentsRoute extends React.Component {
   static manifest = Object.freeze({
     license: {
       type: 'okapi',
       path: 'licenses/licenses/:{id}',
     },
+    linkedAgreements: {
+      type: 'okapi',
+      path: 'licenses/licenses/:{id}/linkedAgreements',
+      params: {
+        sort: 'owner.startDate;desc'
+      },
+      limitParam: 'perPage',
+      perRequest: RECORDS_PER_REQUEST,
+      recordsRequired: '1000',
+      shouldRefresh: preventResourceRefresh({ 'license': ['DELETE'] }),
```

Comment:
I am not sure we need this here. `preventResourceRefresh` here essentially stops from refetching the linkedAgreements on a license deletion. But, we dont allow for a license deletion from an amendment view. So, its probably worth testing it out without it.